app/src/main-process/main.ts

```diff
@@ -737,6 +740,7 @@ function createWindow() {
   window.onClose(() => {
     mainWindow = null
     if (!__DARWIN__ && !preventQuit) {
+      terminateNotificationSystem()
```

Comment:
I think the right place for this would be in `app-window.ts`, in the `'close`' event handler, right after `autoUpdate.removeAllListeners()` which seems to be the place for proper app cleanup.